src/org/labkey/targetedms/outliers/OutlierGenerator.java

```diff
@@ -64,14 +64,31 @@ private String getEachSeriesTypePlotDataSql(int seriesIndex, int id, String sche
             StringBuilder filterClause = new StringBuilder("SampleFileId.ReplicateId in (");
             var intersect = "";
             var selectSql = "(SELECT ReplicateId FROM targetedms.ReplicateAnnotation WHERE ";
+            var escapeQuote = "\'";
             for (AnnotationGroup annotation : annotationGroups)
             {
-                filterClause.append(intersect).append(selectSql).append(" Name='").append(annotation.getName()).append("' AND ( ");
-                var or = "";
-                for (String value : annotation.getValues())
+                filterClause.append(intersect)
+                        .append(selectSql)
+                        .append(" Name=")
+                        .append(escapeQuote)
+                        .append(annotation.getName())
```

Comment:
The concern is that the annotation name or value will have a quote in it. Need to something like:

.append(" Name='").append(annotation.getName().replace("'", "''")).append("'");

Reply:
got it, changed